src/PKSim.Infrastructure/Services/IndividualPropertiesCacheImporter.cs

```diff
@@ -72,6 +72,15 @@ private IndividualPropertiesCache individualPropertiesCacheFrom(string fileFullP
          return individualPropertiesCache;
       }
 
+      private string removeUnitsFromHeader(string header)
```

Comment:
This is not the right place to remove units from header. It is possible to have parameter with unit in name!
For example `Acidic phospholipids [mg/g] - RR`

This one does not end with [] but you see my point .Parameter can be named by the user etc.. so this will potentially fail.
Instead I would suggest to import as is and when trying to match parameter by name, have a fall back option to load a parameter removing potential unit if the parameter was not found